### PR TITLE
Adding FATES EMERALD package

### PR DESCRIPTION
**Pull request by annefou**
- Adding the “Functionally Assembled Terrestrial Ecosystem Simulator” package. It includes Nordic sites.
- License: https://github.com/annefou/staged-recipes/blob/fates-emerald/recipes/fates-emerald/meta.yaml#L49-L54
- Source for FATES EMERALD: https://github.com/NordicESMhub/ctsm/tree/fates_emerald_api

see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fates-emerald/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+sed -i.bak "s/'checkout'/'checkout', '--trust-server-cert'/" ./manage_externals/manic/repository_svn.py
+./manage_externals/checkout_externals
+
+cp .config_files.xml  ${PREFIX}/.
```
Comment (bgruening) [inline recipes/fates-emerald/build.sh:6]: is the dot needed here at the end?

File: recipes/fates-emerald/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - cmake
+    host:
+        - python
+        - mpich
+        - hdf5
+        - netcdf-fortran
+        - svn
```
Comment (bgruening) [inline recipes/fates-emerald/meta.yaml:25]: is svn really needed?

File: recipes/fates-emerald/build.sh (hunk 1 above)
Comment (annefou) [reply recipes/fates-emerald/build.sh]: mostl likely not. I removed it

File: recipes/fates-emerald/meta.yaml (hunk 2 above)
Comment (annefou) [reply recipes/fates-emerald/meta.yaml]: I just tested without and it works. So I removed it. Thanks.

**Review by bgruening — APPROVED** [review]

**Pull request merged**